lib/utils.js

```diff
@@ -45,6 +61,16 @@ function isJQuerySelectExpression(j, node) {
     ;
 }
 
+/**
+ * Transform jQuery specific selectors to a standards-compliant version
+ *
+ * Currently supports:
+ * * `:eq(n)` -> `:nth-child(n+1)`
```

Comment:
This is actually incorrect. `:eq` and `:nth-child` are not equivalent. `$('.foo:eq(2)')` can be replaced by `findAll('.foo')[2]`. The difference is that elements are not necessarily siblings. They can have different parents.